CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ Todos los cambios notables a este proyecto serán documentados en este archivo.
 El formato está basado en [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 y este proyecto adhiere a [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.2.1] - 2019-01-22
+### Added
+- Componente para dibujar QR a partir de occ
```

Comment:
@RodrigoAyala If I right remember the last PR you can draw the QR with the OTT not OCC. I am wrong?

Reply:
@goncafa you're right! I've fixed this in a new commit.